appinventor/components/src/com/google/appinventor/components/runtime/Sound.java

```diff
@@ -298,13 +303,17 @@ public void run() {
   }
 
   private void playAndCheckResult() {
-    streamId = soundPool.play(soundId, VOLUME_FULL, VOLUME_FULL, 0, LOOP_MODE_NO_LOOP,
-        PLAYBACK_RATE_NORMAL);
-  Log.i("Sound", "SoundPool.play returned stream id " + streamId);
-  if (streamId == 0) {
-    form.dispatchErrorOccurredEvent(this, "Play",
-        ErrorMessages.ERROR_UNABLE_TO_PLAY_MEDIA, sourcePath);
-}
+    if (soundPool != null) {
```

Comment:
A slight syntactic change here might be to use a more guard-like syntax where we exit early if soundPool is null.

Reply:
Done.